### DISCUSSION TITLE
part-ii/2021-129/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#81] [Question comments]
Part II 2021 Quantum Information and Computation: Paper 2, Section II, D 

https://questions.tripos.org/part-ii/2021-129/

**Comment by thenewtonium** [comment]
**(a)** – another confusingly-worded question; this is asking for a purely *classical* description of OTP. The ‘additional resource’ referred to is to have *some* method for generating a *random* shared secret.

<details><summary>
<b>(b)</b>

 I initially gave a process that works to generate $\ket{\phi^+}_{AB}$ but with the wrong probability so... *[SPOILERS]*
</summary>

Basically Alice adjoins an ancilla qubit in state $\ket{0}_{A'}$ then applies a unitary operation $U$ with action

$$U\ket{00}_{A'A} = \frac{s}{t}\ket{10}_{A'A} + \sqrt{1-\frac{s^2}{t^2}}\ket{00}_{A'A}$$

and

$$U\ket{01}_{A'A} = \ket{01}_{A'A}$$

then measures the ancilla qubit $A'$, which yields $\ket{1}_{A'}\ket{\phi^+}_{AB}$ with prob. $2s^2$.

The intended method for using this to communicate securely, is for Alice to use the above process to produce (expectedly) $(2s^2)n$ $\ket{\phi^+}_{AB}$ states and communicate publicly to Bob *which* pairs of qubits have been successfully converted, then both measure these qubits in the computational basis to get perfectly correlated outcomes, which can be used as a one-time pad. Importantly, the distribution of outcomes is uniformly random, guaranteeing that the message sent by Alice over the public channel gives no information to Eve.
</details>
